pkgs/applications/audio/goattracker/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv
+, fetchurl
+, unzip
+, makeDesktopItem
+, imagemagick
+, SDL
+}:
+
+let desktopItem = makeDesktopItem {
+  type = "Application";
+  name = "goattracker";
+  desktopName = "GoatTracker 2";
+  genericName = "Music Tracker";
+  exec = "goattrk2";
+  icon = "goattracker";
+  categories = "Multimedia;";
+};
+
+in stdenv.mkDerivation rec {
+  pname = "goattracker";
+  version = "2.75";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/goattracker2/GoatTracker%202/${version}/GoatTracker_${version}.zip";
+    sha256 = "1km97nl7qvk6qc5l5j69wncbm76hf86j47sgzgr968423g0bxxlk";
+  };
+  sourceRoot = "goattrk2/src";
+
+  nativeBuildInputs = [ unzip imagemagick ];
+  buildInputs = [ SDL ];
+
+  # PREFIX gets treated as BINDIR.
+  makeFlags = [ "PREFIX=$(out)/bin/" ];
+
+  # The zip contains some build artifacts.
+  prePatch = "make clean";
+
+  # The destination does not get created automatically.
+  preBuild = "mkdir -p $out/bin";
+
+  # Other files get installed during the build phase.
+  installPhase = ''
+    convert goattrk2.bmp goattracker.png
```

Comment:
The spec requires a png file, but I'm not sure if depending on imagemagick is ok or if I should just drop the icon.